src/components/footer/index.md.njk

```diff
@@ -45,6 +45,19 @@ You can add links to:
 
 Use ‘Privacy’, ‘Accessibility’, ‘Cookies’ and ‘Terms and conditions’ for the link text.
 
+## Use localisation to translate ‘footer’ into other languages
```

Comment:
In other headings on the page, we say 'this component', e.g: 'how to use this component'. Do we want to do something similar here? E.g: Use localisation to translate this component into other languages

Reply:
If this is preferred, then all the headers will need to change to be consistent.

Reply:
Is this section in the right place or would it be better after the 'Adding secondary navigation' section?

Reply:
I _think_ including the name of the component in these headers was to ensure they were unique, when we were still thinking of putting all this guidance on the same page. Now that it's split across pages, we can have all of these headings be the same.